libV2/server/API/metrics/getStorage.js

```diff
@@ -54,7 +54,7 @@ async function getStorage(ctx, params) {
 
     ctx.results.statusCode = 200;
     ctx.results.body = {
-        storageUtilized,
+        storageUtilized: Math.max(storageUtilized, 0),
```

Comment:
Can it go in negative as well?

Reply:
Technically all UTAPI metrics can go negative as it is, at its core, just a system of summing numbers that you give it. This means that if the `+` metrics are missed but the `-` ones aren't you get negatives.